main.go

```diff
@@ -132,7 +136,25 @@ var (
 			Value:     &config.detailsFormat,
 			Default:   "string",
 		},
-		{
+		&sensu.PluginConfigOption[string]{
+			Path:      "alternate-endpoint",
+			Env:       "PAGERDUTY_ALTERNATE_ENDPOINT",
+			Argument:  "alternate-endpoint",
+			Shorthand: "e",
+			Usage:     "The endpoint to use to send the PagerDuty events, can be set with PAGERDUTY_ALTERNATE_ENDPOINT",
+			Value:     &config.alternateEndpoint,
+			Default:   "",
+		},
+		&sensu.PluginConfigOption[uint64]{
+			Path:      "timeout",
+			Env:       "PAGERDUTY_TIMEOUT",
+			Argument:  "timeout",
+			Shorthand: "",
+			Usage:     "The maximum amount of time in seconds to wait for the event to be created, can be set with PAGERDUTY_TIMEOUT",
+			Value:     &config.Timeout,
+			Default:   uint64(30),
+		},
```

Comment:
IMO there should be a way to wait forever, maybe using 0 as a sentinel value?

Reply:
A value of 0 will wait forever, the timeout context is only created if the value is greater than 0. As an example `--timeout=0` will disable the timeout.

I can definitely change the default value to wait forever in a separate PR targeting v2.5.1.